lib/Lower/IO.cpp

```diff
@@ -329,6 +331,24 @@ lowerStringLit(Fortran::lower::AbstractConverter &converter, mlir::Location loc,
   return {buff, len, mlir::Value{}};
 }
 
+/// Pass the body of the FORMAT statement in as if it were a CHARACTER literal
+/// constant. TODO: This feels dumb. Isn't there a front-end method to get this?
```

Comment:
Are you saying that there should be utility routine that  returns the string
```
"(A, F8.3)"
```
for a format like this:
```
 990 format (A, F8.3)
```

Reply:
Yes, exactly. I couldn't find it.